docs/commands.md

```diff
@@ -154,67 +189,17 @@ Optional args:
          2) "28"
       7) 1) (integer) 1548149210
          2) "20"
-127.0.0.1:6379> TS.RANGEBYLABELS area_id=32 id=3 1548149180 1548149210 avg 5
-(empty list or set)
-127.0.0.1:6379> TS.RANGEBYLABELS area_id=32 sensor_id=3 1548149180 1548149210 avg 5
-1) 1) "temperature:3:32"
-   2) 1) 1) "sensor_id"
-         2) "3"
-      2) 1) "area_id"
-         2) "32"
-   3) 1) 1) (integer) 1548149180
-         2) "26.199999999999999"
-      2) 1) (integer) 1548149185
-         2) "27.399999999999999"
-      3) 1) (integer) 1548149190
-         2) "24.800000000000001"
-      4) 1) (integer) 1548149195
-         2) "23.199999999999999"
-      5) 1) (integer) 1548149200
-         2) "25.199999999999999"
-      6) 1) (integer) 1548149205
-         2) "28"
-      7) 1) (integer) 1548149210
-         2) "20"
-127.0.0.1:6379> TS.RANGEBYLABELS area_id=32 sensor_id!=3 1548149180 1548149210 avg 5
-1) 1) "temperature:2:32"
-   2) 1) 1) "sensor_id"
-         2) "2"
-      2) 1) "area_id"
-         2) "32"
-   3) 1) 1) (integer) 1548149180
-         2) "27.600000000000001"
-      2) 1) (integer) 1548149185
-         2) "23.800000000000001"
-      3) 1) (integer) 1548149190
-         2) "24.399999999999999"
-      4) 1) (integer) 1548149195
-         2) "24"
-      5) 1) (integer) 1548149200
-         2) "25.600000000000001"
-      6) 1) (integer) 1548149205
-         2) "25.800000000000001"
-      7) 1) (integer) 1548149210
-         2) "21"
 ```
 
-## TS.INCRBY/TS.DECRBY - Increment the latest value
-```sql
-TS.INCRBY key value [RESET time-bucket]
-```
-or
+## General
+
+### TS.INFO - query the series metadata
 ```sql
-TS.DECRBY key value [RESET time-bucket]
+TS.INFO key
```

Comment:
add a list of metadata available

Reply:
There's an example output with this, isn't this sufficient?